src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -121,6 +120,7 @@ package object csv {
           }
         }
       }
+      }
```

Comment:
Nit: you can remove the open bracket at line 95, and this closing bracket - this is allowed (https://github.com/databricks/scala-style-guide#pattern-matching). Otherwise it looks weird-ish.